### PR TITLE
Update README for 2.37.0/23.08

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  based.
+
+## Known Issues
+
+* Triton uses OpenTelemetry CPP library version, which can cause Triton to [crash](https://github.com/triton-inference-server/server/issues/6202), when OpenTelemetry’s exporter timeouts.
+
+* When using decoupled models, there is a possibility that response order as sent from the backend may not match with the order in which these responses are received by the streaming gRPC client.
+
+* The
+  ["fastertransformer_backend"](https://github.com/triton-inference-server/fastertransformer_backend) is only officially supported for 22.12, though it can be built for Triton container versions up to 23.07.
```
Comment (mc-nv) [inline RELEASE.md:55]: Are we keep supporting "fastertransformet_backend" ?

Comment (tanmayv25) [reply RELEASE.md:55]: No... FT cannot be built with cuda 12.2.1 which is present in 23.08... We can build FT backend till 23.07 though with the latest fix from FT team.

Comment (mc-nv) [reply RELEASE.md:55]: interested as we have that `r23.08` branch in that repo...

Comment (mc-nv) [reply RELEASE.md:55]: I have deleted it

**Review by mc-nv — APPROVED** [review]

**Pull request merged**